### ISSUE TITLE
Migrate code to ES6

### ISSUE THREAD

**Issue by jbphet** [phetsims/rosetta#187]
Since rosetta runs on servers that we control, we've decided it is okay to use more modern ES6 features, such as async/await.  The code should be modified to take advantage of these features to make it more readable and maintainable.

**Comment by zepumph** [comment]
I don't think this needs to be assigned to me, whenever I am in the repo, I am looking for places that would benefit from updating to es6+ practices.

**Comment by ariel-phet** [comment]
Back to you @jbphet - feel free to defer or suggest someone to assign

**Comment by muedli** [comment]
See https://github.com/phetsims/rosetta/projects/1.

**Comment by liam-mulhall** [comment]
- [ ] No instances of `var`.
- [ ] No instances of `function`.
- [ ] All inheritance done via `class` and `extend`.
- [ ] `import` statements instead of `require`.

This should probably be an epic with sub-issues. We also need to make an issue for async/await.

**Comment by liam-mulhall** [comment]
This will be done in the new version of Rosetta.

**Comment by liam-mulhall** [comment]
The new epic is https://github.com/phetsims/rosetta/issues/284. Closing.
